Makefile

```diff
@@ -85,16 +85,19 @@ tests: force
 
 unit-tests: start-kvstores
 	$(QUIET) $(MAKE) -C daemon/ check-bindata
-	$(QUIET) echo "mode: count" > coverage-all.out
+	$(QUIET) echo "mode: count" > coverage-all-tmp.out
 	$(QUIET) echo "mode: count" > coverage.out
 	$(QUIET)$(foreach pkg,$(TESTPKGS),\
 		$(GO) test $(pkg) $(GOTEST_BASE) $(GOTEST_COVER_OPTS) || exit 1; \
```

Comment:
Is there a way we can remove `github.com/cilium/cilium/api/v1` and `github.com/cilium/cilium/pkg/k8s/client` from `TESTPKGS`?